src/Apps/Artist/routes.tsx

```diff
@@ -101,7 +101,7 @@ export const routes: RouteConfig[] = [
       // and not get caught here.
       new Redirect({
         from: "*",
-        to: "/artist/:artist_id",
+        to: "/artist/:artistID",
```

Comment:
Same here. Not sure if something changed on the MP side to account for this, though.